core/src/models/Destination.ts

```diff
@@ -114,6 +115,13 @@ const STATE_TRANSITIONS = [
 @DefaultScope(() => ({
   where: { state: "ready" },
 }))
+@Scopes(() => ({
+  notDraft: {
+    where: {
+      state: { [Op.notIn]: ["draft"] },
+    },
+  },
+}))
```

Comment:
👍 we don't have many scopes in the app yet, but this one is doing to be used a few times.